src/microbe_stage/MicrobeHUD.cs

```diff
@@ -465,6 +467,9 @@ public void EditorButtonPressed()
     {
         GD.Print("Move to editor pressed");
 
+        // To prevent being clicked twice
+        editorButton.MouseFilter = Control.MouseFilterEnum.Ignore;
```

Comment:
Wouldn't it be better to go to "Disabled" state instead?

Reply:
Actually yeah thath makes sense. Changed.